modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -7,6 +7,16 @@ module AppealsApi
   class SupplementalClaim < ApplicationRecord
     include ScStatus
 
+    EVIDENCE = Struct.new(:type, :attributes) do
```

Comment:
I think this should live in our module's `lib` folder as a PORO instead of as a constant. Doesn't feel like a constant if you're calling `new` on it, IMO.

Not a blocker, though. We can play with it later.

Reply:
Yeah, I thought about that too, I just wasn't sure that it was mature enough for it's own file... I tend to think that if we end up doing anything else to it (adding functionality or otherwise), it should be upgraded to its own file.